doc/whats_new.rst

```diff
@@ -47,6 +47,7 @@ Enhancements
 - Add support for anonymizing EDF and BDF files without converting to BrainVision format, by `Austin Hurst`_ (:gh:`636`)
 - Add support for writing FLASH MRI data with :func:`mne_bids.write_anat`, by `Alexandre Gramfort`_ (:gh:`641`)
 - Add interactive data inspector :func:`mne_bids.inspect_dataset`, by `Richard Höchenberger`_ (:gh:`561`)
+- Flat channels and time segments can now be marked using :func:`mne_bids.find_and_mark_flat`, by `Richard Höchenberger`_ (:gh:`649`)
```

Comment:
I would not expose a new function find_and_mark_flat but rather allow an option in inspect_dataset function. Let's start here.